compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -191,18 +194,6 @@ private fun Modifier.onPointerEvent(
     }
 }
 
-private suspend fun PointerInputScope.detectDown(onDown: (Offset) -> Unit) {
-    while (true) {
-        awaitPointerEventScope {
-            val event = awaitPointerEvent(PointerEventPass.Initial)
-            val down = event.changes.find { it.changedToDown() }
-            if (down != null) {
-                onDown(down.position)
-            }
-        }
-    }
-}
-
```

Comment:
Not sure why this was needed, and in any case it's buggy - you're supposed to put the `while` inside the `awaitPointerEventScope`.